<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWidget</class>
 <widget class="QWidget" name="mainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>实验三 网络数据获取与显示</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QtCharts::QChartView" name="chartview"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>选择城市</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboCity">
       <item>
        <property name="text">
         <string>南京</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>北京</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>思南</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>杭州</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>上海</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>选择时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboMonth"/>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>选项</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="cbShowPoint">
          <property name="text">
           <string>显示数据点</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbLegendBold">
          <property name="text">
           <string>图注粗体</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbLegendItalic">
          <property name="text">
           <string>图注斜体</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnLegendAlign">
          <property name="text">
           <string>图注(底)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStart">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>开始</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">QtCharts/QChartView&gt;
#include &lt;QtCharts/chartsnamespace.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
